docs/configuration.v2alpha1.md

```diff
@@ -345,7 +345,7 @@ In the table, `spec.override.nodeAgent.image.name` and `spec.override.nodeAgent.
 | [key].securityContextConstraints.customConfiguration.seLinuxContext.seLinuxOptions.type | Type is a SELinux type label that applies to the container. |
 | [key].securityContextConstraints.customConfiguration.seLinuxContext.seLinuxOptions.user | User is a SELinux user label that applies to the container. |
 | [key].securityContextConstraints.customConfiguration.seLinuxContext.type | Type is the strategy that will dictate what SELinux context is used in the SecurityContext. |
-| [key].securityContextConstraints.customConfiguration.seccompProfiles | SeccompProfiles lists the allowed profiles that may be set for the pod or container's seccomp annotations.  An unset (nil) or empty value means that no profiles may be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When used to generate a value for a pod the first non-wildcard profile will be used as the default. |
+| [key].securityContextConstraints.customConfiguration.seccompProfiles | SeccompProfiles lists the allowed profiles that may be set for the pod or container's seccomp annotations.  An unset (nil) or empty value means that no profiles may be specified by the pod or container.	The wildcard '*' may be used to allow all profiles.  When used to generate a value for a pod the first non-wildcard profile will be used as the default. |
```

Comment:
Similarly, this comes from https://github.com/openshift/api/blob/master/security/v1/types.go#L131

Reply:
Got it, will take the chance to maybe ask there to update.